app/classes/query/base.rb

```diff
@@ -192,44 +203,51 @@ class Query::Base
 
   attr_writer :record
 
-  def self.parameter_declarations
-    { order_by: :string }
+  query_attr(:order_by, :string)
+
+  def self.attribute_types
+    super.symbolize_keys!
```

Comment:
This seems surprising to me.  Why is this needed?

Reply:
It was surprising to me too. Rails returns the `attribute_types` hash with string keys, not "indifferent access", and accessors with symbols fail. It's not the only thing like that and I got tired of the gotcha.